app/views/items/_embargo_form.html.erb

```diff
@@ -1,17 +1,14 @@
-<% if current_user && current_user.is_admin %>
```

Comment:
Indeed. Authorizing the view is bad practice, and actually makes things much harder to test.